guides/appendix/tailwind.md

```diff
@@ -0,0 +1,69 @@
+[Tailwind](https://tailwindcss.com/) is a popular way to use utility-first classes in projects that helps unify large projects using design-tokens-as-code.
```

Comment:
```suggestion
[Tailwind](https://tailwindcss.com/) is a popular way to use utility-first classes in projects that helps unify large projects using design-tokens-as-code.
```

What is a simpler description we can give here? Imagine what you would say to someone sitting next to you if they said “what is tailwind”

Reply:
suggestions? I'm not so great at these sorts of things. 
Like, my two modes are
 - confusingish
 - netflix synopsis (not good, sounds dumb lol)

Reply:
I like your latest update a lot! I'll add any other suggestions there.